frontend/components/BurgerMenu/burgerMenu.module.css

```diff
@@ -3,6 +3,8 @@
 }
 
 .backgraund_burger {
```

Comment:
```suggestion
.background_burger {
```

Reply:
Fixed in commit Fix backgraund -> background